tac/agents/v1/base/game_instance.py

```diff
@@ -362,13 +361,13 @@ def state_after_locks(self, is_seller: bool) -> AgentState:
         state_after_locks = self._agent_state.apply(transactions, self.game_configuration.tx_fee)
         return state_after_locks
 
-    def generate_proposal(self, cfp_services: Dict[str, Union[bool, List]], is_seller: bool) -> List[Description]:
+    def generate_proposal(self, cfp_services: Dict[str, Union[bool, List]], is_seller: bool) -> Optional[List[Description]]:
         """
         Wrap the function which generates proposals from a seller or buyer.
 
         If there are locks as seller, it applies them.
 
-        :param query: the query associated with the cfp.
+        :param cfp_services: the query associated with the cfp.
```

Comment:
change to: the services associated with the cfp

Reply:
Done!